client/me/security-2fa-backup-codes-list/index.jsx

```diff
@@ -162,7 +231,9 @@ module.exports = React.createClass( {
 	},
 
 	renderList: function() {
-		var backupCodes = this.props.backupCodes.length ? this.props.backupCodes : this.getPlaceholders();
+		const backupCodes = this.props.backupCodes.length
+							? this.props.backupCodes
```

Comment:
There's still some excessive indentation here, but we can address this in a follow-up PR.